core/server/data/migrations/init/2-create-fixtures.js

```diff
@@ -10,6 +11,15 @@ module.exports = function insertFixtures(options) {
 
     return Promise.mapSeries(fixtures.models, function (model) {
         logging.info('Model: ' + model.name);
+
+        // The Post model fixtures need a `published_at` date, where at least the seconds
+        // are different, otherwise `prev_post` and `next_post` helpers won't workd with
+        // them.
+        if (model.name === 'Post') {
+            _.forEach(model.entries, function (post, index) {
+                post.published_at = moment().add(index, 'seconds');
```

Comment:
> I can think of a valid case, that a public write access adds a post, which is published without a published date. But for now we can limit the published_at fallback for internal usage.

The fallback in the model layer is already `new Date()` when `published_at` is not set which in general is fine, the problem was specific to fixtures because we were creating multiple posts almost instantaneously and `published_at` only has a resolution of seconds which was breaking prev/next handling.

If it becomes a problem for production sites I'd be inclined to change the date storage format so that we have millisecond resolution instead of resorting to adding seconds to `published_at` values here and there.